SafeSurroundings/SafeSurroundings.Data/Services/InMemoryMeetUpTable.cs

```diff
@@ -46,5 +58,9 @@ public IEnumerable<MeetUp> GetByIDs(IEnumerable<int> ids)
             return listOfMeetups;
         }
 
+        public void Update(int id)
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
Checking if this is needed.

Reply:
Removing the interface as this is only used for testing in memory. Once switching to a database, it wouldn't be needed.